src/notification/notification.component.ts

```diff
@@ -53,7 +53,7 @@ import { of, isObservable, Subject } from "rxjs";
 			</button>
 		</div>
 		<button
-			*ngIf="showClose"
+			*ngIf="notificationObj.showClose"
```

Comment:
This will default to false, _maybe_ something like `notificationObj.showClose === null ? true : notificationObj.showClose`?